internal/databases/sqlserver/sqlserver.go

```diff
@@ -508,9 +509,6 @@ func GetBackupProperties(db *sql.DB,
 		return res, err
 	}
 	defer rows.Close()
```

Comment:
rows.Close() could return error, better to change it to smth like

```suggestion
	defer func() {
	err := rows.Close()
		if err != nil {
			tracelog.ErrorLogger.Printf("failed to close connection %v", err)
		}
	}()
```

Reply:
Sure, added error handling in this defer